ocs_ci/ocs/dr_upgrade.py

```
@@ -181,66 +181,216 @@ def validate_upgrade(self):
-                    self.post_upgrade_data["age"] = pod_obj.get_resource(
-                        resource_name=p.get()["metadata"]["name"], column="AGE"
-                    )
-                    self.post_upgrade_data["pod_status"] = pod_obj.get_resource_status(
-                        resource_name=p.get()["metadata"]["name"]
-                    )
+                # Continue checking other pods, newest might be healthy
+
+            if p_status == "Running":
+                pod_name = p_name
+                pod_status = p_status
+                log.info(f"Found running pod: {pod_name}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)